docs/accessibility-statement.md

```diff
@@ -2,7 +2,7 @@
 title: Accessibility Statement
 ---
 
-_Last updated: April 8, 2019_
+_Last updated: May 11, 2020_
```

Comment:
Maybe we can get rid of this? We now have the "last updated" component:

<img width="1280" alt="screenshot of accessibility statement page showing 'last updated: May 4, 2020'" src="https://user-images.githubusercontent.com/1278991/81616295-17b0db80-9398-11ea-9fec-b4a154c39457.png">